.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+Code Example
+Please create a minimal, reproducible example in an online sandbox (e.g., CodeSandbox, StackBlitz, etc.). Providing code in a sandbox helps us understand and troubleshoot the issue faster.
+
```

Comment:
```suggestion
Code Example
Please create a minimal, reproducible example in an online sandbox and paste a link to it here. We strongly recommend the usage of [Stackblitz](https://stackblitz.com/) as it offers ready-made templates for different frameworks and works great with Tegel. Providing code in a sandbox helps us understand and troubleshoot the issue faster.

Other sandbox tools: CodePen, CodeSandbox


```

Reply:
@ckrook  - I changed this comment as I think Stackblitz would work best for us and our users. I am thinking, how do we make a field where people can paste their links in?

Reply:
@theJohnnyMe  Sounds good to me!